src/nvim/eval.c

```diff
@@ -10209,6 +10207,12 @@ static void f_hasmapto(typval_T *argvars, typval_T *rettv)
     rettv->vval.v_number = FALSE;
 }
 
+/// `hastabdir()` function
+static void f_hastabdir(typval_T *argvars, typval_T *rettv)
+{
+  rettv->vval.v_number = (curtab->localdir != NULL);
```

Comment:
Yes, but not in this feature request. I wanted to get `:tcd` to the same level as `:cd` and `:lcd` and everything from now on will affect all three.